src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/configuration/AppProperties.kt

```diff
@@ -6,8 +6,10 @@ import org.springframework.boot.context.properties.ConstructorBinding
 @ConstructorBinding
 @ConfigurationProperties("application")
 data class AppProperties(
-        val enhetsregisteretUrl: String,
-        val organizationCatalogueUrl: String,
-        val municipalityUrl: String,
-        val organizationDomainsUrl: String) {
-}
\ No newline at end of file
+    val enhetsregisteretUrl: String,
+    val enhetsregisteretProxyUrl: String = enhetsregisteretUrl,
```

Comment:
Har ikke application.properties en default verdi for proxy hvis miljøvariabelen ikke er satt?

Dvs at den vil aldri være null -> aldri bli satt til enhetsregisteretUrl og alltid prøve å koble seg mot nginx-enhetsregisteret-proxy-mock

Reply:
bra funn!, skal fjerne den!